libr/anal/p/anal_mips_gnu.c

```diff
@@ -1383,7 +1383,7 @@ static int mips_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *b, int len, R
 				case 17: //bal  bgezal
 					if (rs==0) {
 						op->jump = addr+(imm<<2)+4;
-						snprintf (insn.i_reg.jump, REG_BUF_MAX, "0x%lx", op->jump) ;
+						snprintf (insn.i_reg.jump, REG_BUF_MAX, "0x%"PFMT64x, op->jump) ;
```

Comment:
pfmt64x is fine, but jfyi: %lx is long int, and %llx (is not portable) and for ut64

Reply:
Sorry, it's so late. I was on vacation the other day.
We should use %llx to ensure that an 8-byte address is generated on both 32-bit and 64-bit systems.